.github/workflows/repatriate.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           this_branch=release/${this_ver}
           echo "::set-output name=this_branch::${this_branch}"
 
-          next_ver=$(git for-each-ref --format='%(refname:lstrip=4)' 'refs/remotes/origin/release/v*' | sort --version-sort | grep -A1 -Fx "$this_ver" | sed 1d)
+          next_ver=$(git for-each-ref --format='%(refname:lstrip=4)' 'refs/remotes/origin/release/v*' | sort --version-sort | grep -vFx -e 'v3.0' -e 'v3.1' | grep -A1 -Fx "$this_ver" | sed 1d)
```

Comment:
Would we have to keep updating this line for new 3.y releases?

Reply:
Presumably no. Assuming that 3.2 will have feature parity with 2.4, everything that goes in to 2.4 will be repatriatable to 3.2. It may reduce the tedium to add new releases here, but it wouldn't be problematic if we don't. If we do a 2.5, we may have to re-assess this list, but please let's not do a 2.5.